### PR TITLE
slider component added

### PR DESCRIPTION
**Pull request by vaibhav18matere**
slider

### CODE REVIEW

**Review by harshkotkar5557 — APPROVED** [review]
clean code, you can merge it

File: components/slider/slider.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.slider-range{
+    min-width: 350px;
```
Comment (amarphule) [inline components/slider/slider.css:2]: you can use `rem` unit

**Review by tewarimohit — APPROVED** [review]
Looks good to merge....

**Review by amarphule — APPROVED** [review]
Looks good to me

**Review by praveenreddy1798 — APPROVED** [review]

**Pull request merged**